mp/importer/batcher.py

```diff
@@ -0,0 +1,25 @@
+
+def run_in_batches(iterable, end_batch_callback, batch_size, start_batch, max_batches):					# arguments passed through "options"
+    c = 0
+    ended = False
+    for k in iterable:
+        c = c+1			
```

Comment:
gonna get nitpicky here now about PEP-8: please put spaces on both sides of the +